requirements.txt

```diff
@@ -1,6 +1,6 @@
-securesystemslib==0.10.10
+securesystemslib>=0.10.10
```

Comment:
Is the `>=` needed to catch breaking changes during development?  It might be good to explain in the pull request summary.

Also, is it okay to still pin securesystemslib to 0.10.10 in `setup.py?`

Reply:
I'll elaborate on this. Also thanks for the reminder on the 0.10.10 in setup.py. Sorry for the late response!